python/README.md

```diff
@@ -2,8 +2,84 @@
 
 ### Installation
 
-Inside a virtualenv, run:
+#### Python 3.6
+
+Energuide API - ETL is tested to run on Python 3.6. To begin, you will need to have Python 3.6 installed.
+
+We recommend `pyenv` as an easy way to easily install and switch between any versions of Python that you need.
+
+See https://github.com/pyenv/pyenv#installation for pyenv installation instructions.
+
+#### MongoDB
+
+To run the ETL, you need to be able to access a MongoDB instance. For instructions on installing one locally, see https://docs.mongodb.com/manual/administration/install-community/
+
+#### Virtualenv
+
+Installing Python applications in a `virtualenv` is considered best practice. To do so, run:
+```
+python3 -m venv env
+source env/bin/activate
+```
+This will create a new virtualenv in a  folder called `env`, and activate the virutalenv. To deactivate the virtualenv, run `deactivate`
+
+#### Installing the app
+
+Inside an activated virtualenv, and from the python folder of the project, run:
 ```
 pip install -r requirements.txt
 pip install -e .
 ```
+
+#### Running the app
+
+The ETL application is accessed from the `energuide` CLI. Run `energuide --help` for help.
+
+There is currently a single command for energuide: 
+```
+energuide load --filename /path/to/file
+```
+
+A sample file is included for demonstration purposes at `./tests/sample.csv`
```

Comment:
We should likely advise them to use randomized_energuide_data.csv instead as it more accurately reflects the real data.